metecho/api/models.py

```diff
@@ -38,7 +38,7 @@
 from .validators import validate_unicode_branch
 
 ORG_TYPES = Choices("Production", "Scratch", "Sandbox", "Developer")
-SCRATCH_ORG_TYPES = Choices("Dev", "QA")
+SCRATCH_ORG_TYPES = Choices("Dev", "QA", "Beta", "Release")
```

Comment:
@wlonk We should discuss this, probably including additional clarification from @davisagli. My understanding is that we are not changing the front-end user-facing UX around there being 0 or 1 "Dev Org" and 0 or 1 "Test Org" for each Task. What we are changing is _only_ the flow that those orgs run on creation.

If that's the case, then this change seems like we're overloading the meaning of `SCRATCH_ORG_TYPES`. We still want a distinction between Dev/Test org for the task, but we _also_ want to know what flow new orgs should run when created under that Task.

Since this is a Task-level setting, I think we need a new field on the Task model to track which flow-type is selected (a la @duggiemitchell's question in Slack: https://oddbird.slack.com/archives/CH6EN9NMA/p1590514112005100).

Reply:
@jgerigmeyer you're correct that the org choice being added for this story (affecting how the org is built) is different from the Dev/Test org types shown in the UI (which is more about the org's role in the development -> testing process)

But the new field on task should store the name of the org config that was selected by the user, not the flow. The flow should be determined at runtime from the org config as follows:
1. Use the stored org config name to get the actual org config object (`project_config.keychain.get_org(org_name)`
2. If org_config.setup_flow is not None, run the flow with this name (at the moment this never exists, but later it will)
3. Otherwise pick the setup flow using the logic from my proposal in Trello: `use a hardcoded mapping for which flow to run based on the org config name (dev or feature -> dev_org, qa -> qa_org, beta -> install_beta, release -> install_prod).`